### PR TITLE
Updates to 0.13 typedefs

### PR DESCRIPTION
**Pull request by rmccallum81**
## :memo: Description

Type definitions are updated.

### :dart: Relevant issues
Based off comments in PR #318 

### :gem: Type of change

- [x] Type definitions

## :vertical_traffic_light: How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Run existing tests
- [x] Build test service and run

## :checkered_flag: Checklist:

- [x] I have performed a self-review of my own code
- [x] **New and existing unit tests pass locally with my changes**

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -160,21 +168,20 @@ declare namespace Moleculer {
...
 		waitForServices(serviceNames: string | Array<string>, timeout?: number, interval?: number): Bluebird<void>;
 
-		_init: () => void;
-		_start: () => Bluebird<void>;
-		_stop: () => Bluebird<void>;
-		[name: string]: any;
+		_init(): void;
+		_start(): Bluebird<void>;
+		_stop(): Bluebird<void>;
 	}
```
Comment (icebob) [inline index.d.ts]: Why did you remove this `[name: string]: any`? The schema contains any other properties, because mixins can handle them.

Comment (rmccallum81) [reply index.d.ts]: Sorry was looking at the effect of removing it as ES6 classes shouldn't have this type of index definition. It removes much of the type safety of the class. I forgot to put it back in after my testing.

**Comment by icebob** [comment]
Thank you!

**Comment by icebob** [comment]
Something wrong: https://travis-ci.org/moleculerjs/moleculer/jobs/402546418#L2412
